bpf/kepler.bpf.c

```diff
@@ -265,27 +34,21 @@ int kepler_irq_trace(u64 *ctx)
 SEC("fexit/mark_page_accessed")
 int kepler_read_page_trace(void *ctx)
 {
-	u32 curr_tgid;
-	struct process_metrics_t *process_metrics;
+	u32 curr_pid;
 
-	curr_tgid = bpf_get_current_pid_tgid() >> 32;
-	process_metrics = bpf_map_lookup_elem(&processes, &curr_tgid);
-	if (process_metrics)
-		process_metrics->page_cache_hit++;
+	curr_pid = bpf_get_current_pid_tgid();
+	do_page_cache_hit_increment(curr_pid);
 	return 0;
 }
 
 // count write page cache
 SEC("tp/writeback_dirty_folio")
 int kepler_write_page_trace(void *ctx)
 {
-	u32 curr_tgid;
-	struct process_metrics_t *process_metrics;
+	u32 curr_pid;
 
-	curr_tgid = bpf_get_current_pid_tgid() >> 32;
-	process_metrics = bpf_map_lookup_elem(&processes, &curr_tgid);
-	if (process_metrics)
-		process_metrics->page_cache_hit++;
+	curr_pid = bpf_get_current_pid_tgid();
```

Comment:
this almost always get me .. I wonder why this its named `_pid_tgid` instead of `_tgid_pid`